### PR TITLE
Adds preference for seeing LOOC from ghosts

### PR DESCRIPTION
**Pull request by Doxxmedearly**
Adds a toggle for players who do not want to see LOOC messages from observers/ghosts. Added in the preferences tab as well as character setup.

### CODE REVIEW

**Comment by Doxxmedearly** [comment]
!review

**Review by Geevies — APPROVED** [review]

**Review by Arrow768 — CHANGES_REQUESTED** [review]
You are not accounting for LOOC messages sent by holders that are observing.

**Review by JohnWildkins — APPROVED** [review]

**Comment by Doxxmedearly** [comment]
Regular LOOC was not accounting for admins, either.
Added admin LOOC as discussed since tying to holders is kinda sketch.

**Review by Alberyk — APPROVED** [review]

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -1339,3 +1341,66 @@ var/list/admin_verbs_cciaa = list(
...
+				messagemobs += M
+				continue
+		if(get_turf(M) in messageturfs)
+			messagemobs += M
+
+	var/display_name = source.key
+	if(holder && holder.fakekey)
+		display_name = holder.fakekey
+
+	msg = process_chat_markup(msg, list("*"))
```
Comment (Arrow768) [inline code/modules/admin/admin_verbs.dm:1389]: here is the second time you are calling it

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**